sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/CompactBlockProcessor.kt

```diff
@@ -946,8 +946,8 @@ class CompactBlockProcessor internal constructor(
             ?.toHexReversed()
         val prevHash = repository.findBlockHash(errorHeight)?.toHexReversed()
 
-        val compactBlock = downloader.compactBlockRepository.findCompactBlock(errorHeight + 1)
-        val expectedPrevHash = compactBlock?.prevHash?.toByteArray()?.toHexReversed()
+        // val compactBlock = downloader.compactBlockRepository.findCompactBlock(errorHeight + 1)
+        val expectedPrevHash = "" // compactBlock?.prevHash?.toByteArray()?.toHexReversed()
```

Comment:
Seems like it's because we don't have `prevHash` field in the new `JniBlockMeta` object and we also can't get it from `DerivedDataRepository`. I just removed the `prevHash` logging in this case.